Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -144,6 +153,47 @@ private void UpdateShuttleCargoConsoles(CargoShuttleComponent component)
         }
     }
 
+    private void UpdatePalletConsoleInterface(EntityUid uid, CargoPalletConsoleComponent component)
+    {
+        var bui = _uiSystem.GetUi(component.Owner, CargoPalletConsoleUiKey.Sale);
+
+        if (!TryComp<TransformComponent>(uid, out var xform) || xform.GridUid == null)
```

Comment:
Transform() proxy method (cast by pattern match here so you dont have to re-declare a griduid local var) (also grey the button if this condition fails)